test/remote/gateways/remote_payeezy_test.rb

```diff
@@ -33,6 +33,14 @@ def setup
       initiator: 'MERCHANT',
       auth_type_override: 'A'
     }
+    @options_standardized_stored_credentials = {
+      stored_credential: {
+        network_transaction_id: 'abc123', # Not checked if initial_transaction == true; not valid if initial_transaction == false.
```

Comment:
For my own curiosity: Seeing the comment, does this mean that it needs to be present if `initial_transaction == true`, but the value isn't checked?